src/components/ha-camera-stream.ts

```diff
@@ -0,0 +1,153 @@
+import { property, UpdatingElement, PropertyValues } from "lit-element";
+
+import computeStateName from "../common/entity/compute_state_name";
+import { HomeAssistant, CameraEntity } from "../types";
+import { fireEvent } from "../common/dom/fire_event";
+import { fetchStreamUrl, computeMJPEGStreamUrl } from "../data/camera";
+
+type HLSModule = typeof import("hls.js");
+
+class HaCameraStream extends UpdatingElement {
+  @property() public hass?: HomeAssistant;
+  @property() public stateObj?: CameraEntity;
+  @property() public showControls: boolean = false;
```

Comment:
```suggestion
  @property({type: Boolean}) public showControls = false;
```

TypeScript type is not needed with a default vaule, it will infer.
Lit type is important, so existence of the attribute will set the value to true.